lib/elaborate/elaborateReferences.ts

```diff
@@ -261,19 +261,37 @@ export class ElaborateNames {
     const packages = lastPrefix.definitions.filter(def => def instanceof O.OPackage) as O.OPackage[];
     // previous token is pkg inst or interface pkg -> find stuff from the uninstantiated package
     const pkgInstantiations = lastPrefix.definitions.filter(def => def instanceof O.OPackageInstantiation || def instanceof O.OInterfacePackage) as (O.OPackageInstantiation | O.OInterfacePackage)[];
+    for (const pkgInstantiation of pkgInstantiations) {
+      for (const ref of pkgInstantiation.uninstantiatedPackage) {
+        this.elaborate(ref);
+      }
+    }
     packages.push(...pkgInstantiations.flatMap(inst => inst.uninstantiatedPackage[inst.uninstantiatedPackage.length - 1]!.definitions).filter(ref => ref instanceof O.OPackage) as O.OPackage[]);
     for (const pkg of packages) {
+
       for (const decl of pkg.declarations) {
-        if (decl.lexerToken !== undefined && decl.lexerToken.getLText() === name.nameToken.getLText()) {
+        if (decl.lexerToken !== undefined && (decl.lexerToken.getLText() === name.nameToken.getLText() || name.nameToken.getLText() === 'all')) {
```

Comment:
`all` should only be linked when `name.parent` is instanceof OUseClause. Otherwise, the linter will accept `.all` after every name.